product-mini/platforms/windows/CMakeLists.txt

```diff
@@ -151,6 +152,11 @@ set_target_properties (libiwasm PROPERTIES OUTPUT_NAME libiwasm)
 
 target_link_libraries (libiwasm ${LLVM_AVAILABLE_LIBS} ${UV_A_LIBS})
 
+
 if (MINGW)
   target_link_libraries (libiwasm ws2_32)
 endif ()
+
+if (WIN32)
```

Comment:
is it possible to merge MING with WIN32? Are both being used when you build on windows?